src/ts/uimanager.ts

```diff
@@ -395,10 +395,10 @@ export class UIManager {
      * undesirable at this time. */
     this.uiContainerElement.append(dom);
 
-    // Some components initialize their state on SourceLoaded. When the UI is loaded after the source is already loaded,
-    // they will never receive the event so we fire it from here in such cases.
+    // Some components initialized their state on SourceLoaded. When the UI is loaded after a source was loaded,
```

Comment:
"Some components initialized their state on SourceLoaded" is unnecessary. We don't need to document how something worked on an old version :)